fiftyone/factory/repos/execution_store.py

```diff
@@ -34,23 +35,42 @@ def __init__(self, collection: Collection, dataset_id: ObjectId = None):
         self._collection = collection
         self._dataset_id = dataset_id
 
-    def create_store(self, store_name) -> StoreDocument:
+    def create_store(
```

Comment:
I saw a PR where @imanjra plans to change this to `def store()` so just a heads up in case a conflict arises